src/Faithlife.Build/BuildRunner.cs

```diff
@@ -42,11 +43,38 @@ public static int Execute(string[] args, Action<BuildApp> initialize)
 
 			commandLineApp.OnExecute(() =>
 			{
-				var bullseyeArgs = targetsArgument.Values.ToList();
+				var bullseyeArgs = targetsArgument.Values.WhereNotNull().ToList();
 
 				if (bullseyeArgs.Count == 0 && buildApp.Targets.Any(x => x.Name == c_defaultTarget))
 					bullseyeArgs.Add(c_defaultTarget);
 
+				var skipDependencies = skipDependenciesFlag.Value;
+				if (skipOption.Value is not null && !skipDependencies)
+				{
+					var skipTargetNames = new HashSet<string>(skipOption.Value.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries));
+					skipDependencies = true;
+
+					var targetNames = new HashSet<string>(bullseyeArgs);
+					foreach (var targetName in bullseyeArgs.ToList())
```

Comment:
This is a bit subtle (having `AddTargetDependencies` capture _and mutate_ this collection); I might have moved it to `var originalBullseyeArgs = bullseyeArgs.ToList();` for clarity. Or not mutated the captured local.

Reply:
It's true. Since you noticed I was being lazy, I'll fix it up.

Reply:
Reorganized the code. Any better?